### PR TITLE
Change json fields to all lower case for consistency

### LINKED ISSUE #261: Difference between implementation and documentation of GetVersion RPC method
**Describe the bug**
Implementation of [`GetVersion`](https://github.com/neo-project/neo-modules/blob/master/src/RpcServer/RpcServer.Node.cs#L60) RPC method assumes lover-cased underscored JSON fields `tcp_port`, `ws_port` and `user_agent`. Documentation [example](https://docs.neo.org/v3/docs/en-us/reference/rpc/latest-version/api/getversion.html) differs from that and describes `tcpPort`, `wsPort` and `useragent`.

**Expected behavior**
Documentation should match implementation.

### PR DESCRIPTION
**Pull request by joeqian10**
Close #261

### CODE REVIEW

**Comment by joeqian10** [comment]
Need to update dependencies to pass UTs.

File: src/LevelDBStore/Plugins/Storage/Snapshot.cs [hunk 1]
```diff
@@ -35,9 +36,9 @@ public void Dispose()
             snapshot.Dispose();
         }
 
-        public IEnumerable<(byte[] Key, byte[] Value)> Find(byte table, byte[] prefix)
+        public IEnumerable<(byte[] Key, byte[] Value)> Seek(byte table, byte[] keyOrPrefix, SeekDirection direction = SeekDirection.Forward)
         {
-            return db.Find(options, Helper.CreateKey(table, prefix), (k, v) => (k[1..], v));
+            return db.Find(options, Helper.CreateKey(table, keyOrPrefix), (k, v) => (k[1..], v));
```
Comment (shargon) [inline src/LevelDBStore/Plugins/Storage/Snapshot.cs:41]: `direction` is not working

File: src/LevelDBStore/Plugins/Storage/Store.cs [hunk 2]
```diff
@@ -49,7 +50,7 @@ public void Dispose()
             db.Dispose();
         }
 
-        public IEnumerable<(byte[], byte[])> Find(byte table, byte[] prefix)
+        public IEnumerable<(byte[], byte[])> Seek(byte table, byte[] prefix, SeekDirection direction = SeekDirection.Forward)
         {
             return db.Find(ReadOptions.Default, Helper.CreateKey(table, prefix), (k, v) => (k[1..], v));
```
Comment (shargon) [inline src/LevelDBStore/Plugins/Storage/Store.cs:55]: `direction` is not working

File: src/LevelDBStore/Plugins/Storage/Snapshot.cs (hunk 1 above)
Comment (joeqian10) [reply src/LevelDBStore/Plugins/Storage/Snapshot.cs:41]: I'm trying to fix this.

Comment (joeqian10) [reply src/LevelDBStore/Plugins/Storage/Snapshot.cs:41]: @shargon `ByteArrayComparer` is internal, which I cannot use it here.

Comment (joeqian10) [reply src/LevelDBStore/Plugins/Storage/Snapshot.cs:41]: Could you help to fix this?

Comment (Tommo-L) [reply src/LevelDBStore/Plugins/Storage/Snapshot.cs:41]: Waiting for https://github.com/neo-project/neo-modules/pull/279

**Comment by ProDog** [comment]
I tested an exception, like caused by LevelDB:

image

File: src/RpcServer/RpcServer.SmartContract.cs [hunk 3]
```diff
@@ -96,27 +92,5 @@ private JObject InvokeScript(JArray _params)
             CheckWitnessHashes checkWitnessHashes = _params.Count >= 2 ? new CheckWitnessHashes(((JArray)_params[1]).Select(u => UInt160.Parse(u.AsString())).ToArray()) : null;
             return GetInvokeResult(script, checkWitnessHashes);
         }
-
-        [RpcMethod]
-        private JObject GetUnclaimedGas(JArray _params)
```
Comment (shargon) [inline src/RpcServer/RpcServer.SmartContract.cs:101]: This method was removed?

Comment (Tommo-L) [reply src/RpcServer/RpcServer.SmartContract.cs:101]: @joeqian10  Move to https://github.com/neo-project/neo-modules/pull/273?

**Comment by ProDog** [comment]
> I tested an exception, like caused by LevelDB:
> 
Fixed in https://github.com/neo-project/neo-modules/pull/281.

**Comment by ProDog** [comment]
plz update this branch @joeqian10 .

**Pull request merged**